package.json

```diff
@@ -12,7 +12,7 @@
     "lint-fix": "yarn lint -w",
     "lint-watch": "yarn lint -m",
     "pregenerate": "node utils/pregenerate.js",
-    "_dev-only": "nuxt --port 4000",
+    "dev-only": "nuxt --port 4000",
```

Comment:
Please note that updating the above would cause the `dev` script to break. Please update `dev` script to 
`"dev": "npm-run-all --parallel dev-only _lint-watch-fix _api-watch pregenerate",`. (i.e. replace the usages of `_dev-only` with `dev-only`)